### DISCUSSION TITLE
macOS big sur 11.2.2 Homebrew can't install ntfs-3g

### DISCUSSION THREAD

**Discussion by Evkva** [Homebrew/discussions#1164] [Everyday usage]
![截屏2021-03-30 下午4 46 26](https://user-images.githubusercontent.com/32220583/112981130-55c01500-918d-11eb-87fd-8529a9a51fc0.png)

My CPU is intel,but

![截屏2021-03-30 下午7 25 36](https://user-images.githubusercontent.com/32220583/112981411-ba7b6f80-918d-11eb-9648-b71617b7a304.png)

please update

**Comment by SMillerDev** [comment]
See https://docs.brew.sh/Updating-Software-in-Homebrew
